smriprep/workflows/anatomical.py

```diff
@@ -335,8 +335,8 @@ def init_anat_preproc_wf(
             (('t1w', fix_multi_T1w_source_name), 'inputnode.source_file')]),
         (anat_template_wf, anat_reports_wf, [
             ('outputnode.out_report', 'inputnode.t1_conform_report')]),
-        (anat_template_wf, seg_rpt, [
-            ('outputnode.t1_template', 'in_file')]),
+        (brain_extraction_wf, seg_rpt, [
+            (('outputnode.bias_corrected', _pop), 'in_file')]),
```

Comment:
Could you clarify why `_pop` is needed here? Shouldn't there only be one bias-corrected file?